configure.ac

```diff
@@ -178,7 +178,7 @@ fi
 ##       compatibility with the legacy buildsystem.
 ##
 if test "x$CXXFLAGS_overridden" = "xno"; then
-  CXXFLAGS="$CXXFLAGS -Wall -Wextra -Wformat -Wformat-security -Wno-unused-parameter -Wno-self-assign"
```

Comment:
Wasn't this added recently? Reference https://github.com/bitcoin/bitcoin/pull/6744